### PR TITLE
Created card header and images

### PR DESCRIPTION
**Pull request by sanu29**
image
image
image
image

### CODE REVIEW

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
+
+.card{
+    display: flex;
+    width:fit-content;
+    flex-direction: column;
+    margin: 3rem;
+    background-color: #FFFFFF;
```
Comment (zoef-yt) [inline components/card.css]: use var for colour

File: components/card.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    padding: 1rem 0 1rem 0;
+    text-align: center;
+}
+.card-secondary{
+    padding: 0 1rem 0.5rem 1rem;
+    font-weight: 400;
+    line-height: 1.25rem;
+    font-size: 0.88rem;
+}
+
```
Comment (developer-anand) [inline components/card.css]: remove spacing.

File: components/card.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+.card-icon-dismiss{
+    color: var(--danger);
+    top: 0rem;
+    right:-5px;
+    padding: 0.4rem; 
+    margin: 1rem;  
+    background-color: #FFFFFF;
```
Comment (zoef-yt) [inline components/card.css]: use var for colour

File: components/header.css [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    padding: 0.5rem;
+    color: #fff;
+    border-radius:5px;
+    margin: 1rem 0rem 1rem 0;
+}
+.header-main-text{
+    width: fit-content;
+    margin:0 1.5rem 0 0.5rem ;;
+}
+.header-icon{
```
Comment (developer-anand) [inline components/header.css:17]: Be consistent with spacing.

File: components/header.css [hunk 5]
```diff
@@ -0,0 +1,45 @@
+
+.header{
+    display: flex;
+    align-items: center;  
+    justify-content: start;
+    width: 100vw;
+    background-color: var(--primary-color);
+    padding: 0.5rem;
+    color: #fff;
```
Comment (zoef-yt) [inline components/header.css]: use var for colour

**Review by developer-anand — APPROVED** [review]
Good to merge with changes.

File: components/header.css [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    justify-content: end;
+}
+
+.header-short{
+    width: 8rem;
+    display: flex;
+    flex-direction: row;
+    background-color: var(--primary-color);
+    padding: 0.5rem 1rem  0.5rem 0  ;
+    color: #fff;
```
Comment (zoef-yt) [inline components/header.css:37]: use var for colour

File: components/header.css [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+    margin: 1rem 0rem 1rem 0;
```
Comment (zoef-yt) [inline components/header.css]: use margin: 1rem 0;

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by zoef-yt — APPROVED** [review]
Nice work. Good to merge.

**Pull request merged**